sync/04-docker-composer.sh

```diff
@@ -30,13 +32,6 @@ cp -r "$COMPOSER_WORK_DIR"/vendor "$PWD"/dist
 # composer install
 COMPOSER_WORK_DIR="$PWD"/dist
 
-# Run without composer-merge plugin to install using only composer.lock
```

Comment:
I think we can not remove this actually; running `install` without the merge plugin seems to install lots of packages that are missing otherwise, for some reason

Reply:
Wait I just realized I have to not only remove `dist/vendor` but all the other ones too ... testing one more time